webpack/assets/javascripts/services/ChartService.consts.js

```diff
@@ -14,9 +15,11 @@ export const donutChartConfig = {
     label: { show: false },
   },
   data: {
-    type: 'donut',
     columns: [],
   },
+  color: {
+    pattern: ['#0088ce', '#cc0000', '#ec7a08', '#3f9c35', '#005c66', 'f9d67a'],
```

Comment:
just making sure this is intended.. the colors are different from the current color set.
before
![image](https://user-images.githubusercontent.com/20167/36474991-fe024512-1701-11e8-896d-d3e8baf75427.png)


now
![image](https://user-images.githubusercontent.com/20167/36474970-f351a7e8-1701-11e8-9338-4da52e311e36.png)

Reply:
this color pattern is taken from `patternfly` with @Rohoover  approval.
still, we need to re-design charts with more then 5 categories, due to patternfly, this chart type supports up to 5 categories.